transformer/events_creator.go

```diff
@@ -64,7 +64,7 @@ func NewTransformer(log *logp.Logger, cd CommonDataInterface, index string) Tran
 	}
 }
 
-func (t *Transformer) CreateBeatEvents(ctx context.Context, eventData evaluator.EventData) ([]beat.Event, error) {
+func (t *Transformer) CreateBeatEvents(_ context.Context, eventData evaluator.EventData) ([]beat.Event, error) {
```

Comment:
Let's remove it if not in use.

Reply:
it implements a function interface, so we can't remove it, but since we don't use it it's common to set it with underscore